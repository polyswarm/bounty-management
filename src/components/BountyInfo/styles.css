.Bounty-Info {
  height: 100%;
  position: relative;
  width: 100%;
}

.Bounty-Info-Container {
  display: grid;
  grid-template-columns: 1fr 1fr;
  height: 100%;
  padding: 20px;
}

.Bounty-Info-Files {
  grid-column: 0;
}
