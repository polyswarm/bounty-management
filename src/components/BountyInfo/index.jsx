// Vendor imports
import React, { Component } from 'react';
import PropTypes from 'prop-types';
// Bounty imports
import FileList from '../FileList';
// Component Imports
import './styles.css';

class BountyInfo extends Component {

  render() {
    // const { props: { bounty } } = this;
    // const files = bounty.artifacts;
    // const assertions = bounty.assertions;
    const files = [];
    return (
      <div className='Bounty-Info'>
        <div className='Bounty-Info-Container'>
          <FileList className='Bounty-Info-Files' files={files} immutable />
        </div>
      </div>
    );
  }
}

BountyInfo.propTypes = {
  bounty: PropTypes.object.isRequired,
};
export default BountyInfo;
