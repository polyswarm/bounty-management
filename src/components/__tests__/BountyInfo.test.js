import React from 'react';
import {render} from 'enzyme';
import {renderToJson} from 'enzyme-to-json';
import BountyInfo from '../BountyInfo';

it('renders without crashing', () => {
  const wrapper = render(<BountyInfo />);
  expect(renderToJson(wrapper)).toMatchSnapshot();
});
