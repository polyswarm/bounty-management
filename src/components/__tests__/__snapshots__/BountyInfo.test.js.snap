// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  class="Bounty-Info"
>
  <div
    class="Bounty-Info-Container"
  >
    <div
      class="File-List"
    >
      <header
        class="File-List-Header"
      >
        <p
          class="File-List-Title"
        >
          0 Files
        </p>
      </header>
      <ul
        class="List"
      >
        
      </ul>
    </div>
  </div>
</div>
`;
